src/components/Checkout/BillingInfo/BillingInfo.js

```diff
@@ -95,27 +90,32 @@ class BillingInfo extends React.Component {
             <label htmlFor="address2">address 2</label>
             <input type="text" name={CN} className={`${CN}__input`} />
           </div>
-          <div className={`${CN}__input-wrapper`}>
-            <p>country</p>
-            <Dropdown
-              menuOptions={dropdownOptions.billingInfo.country}
-              value="Choose country..."
-              onDropdownChange={onDropdownChange}
-              className={`${CN}__dropdown`}
-            />
+          <div className={`${CN}-selector-wrapper`}>
+            <h5 className={`${CN}-selector__title`}>country</h5>
+            <div className={`${CN}-selector-container`}>
+              <CountryDropdown
+                class={`${CN}-selector-container__selector`}
+                value={country}
+                onChange={(val) => this.selectCountry(val)}
```

Comment:
You can use here, it prevent unnecessary creation of callbacks for every render
```suggestion
                onChange={this.selectCountry}
```